### PR TITLE
feat: Implemented OnBoarding Screens

### PR DESCRIPTION
**Pull request by s-ayush2903**
* Implemented onBoarding(intro) screens
* The last slide of intro screen contains a set of permissions that are mandatory, mandatory in the sense that user will have to grant permissions in order to further use the app. Check this use case when the user denies the permissions, there is a _TODO_ in the concerned activity, see that
* Changed the intro Lib provider, the current one being used now is [AppIntro](https://github.com/AppIntro/AppIntro)

**Screenshots**
|Slide#1|Slide#2|
|---------|-----------|
|Screenshot_20200809-125740_LocerAndroid|ss2

|Slide#3|Slide#4(Permissions Slide)|
|-------------------|----------------------|
|ss3|Screenshot_20200809-130222_LocerAndroid

The slide#3 is distorted here because i'm not very well proficient in attaching screenshots in md format, but it behaves as the others do in the physical device

### CODE REVIEW

File: app/src/main/res/drawable/permits.xml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<vector android:autoMirrored="true" android:height="727.7781dp"
```
Comment (s-ayush2903) [inline app/src/main/res/drawable/permits.xml:1]: This is the vector drawable used for the image being shown in the permissions slide

**Comment by s-ayush2903** [comment]
Here is the working introScreen::
intro-screencast

File: app/src/main/res/drawable/permits.xml (hunk 1 above)
Comment (Raushansingh19) [reply app/src/main/res/drawable/permits.xml:1]: nice we are delivering food + groceries and laundry so we can add all of them in one slide .
on 2nd we can add this.

**Comment by Raushansingh19** [comment]
i will send you some templates so we can add this

**Pull request merged**